docs/development/ide-setup/intellij-setup.md

```diff
@@ -10,10 +10,12 @@ Setting up *IntelliJ* is pretty simple.  There are a few plugins to install with
       ](https://user-images.githubusercontent.com/12397753/62746114-07cc2b80-ba03-11e9-9ac0-0b1e6e1e8788.png)
   1. *checkstyle-IDEA* [plugin](https://github.com/jshiell/checkstyle-idea)
         1. after install finish configuration in: **Other Settings > Checkstyle**
-        1. select checkstyle file, set checkstyle version, set to scan all sources
+            1. load checkstyle file by clicking on the "plus" and navigating to the file or downloading it from [the repository](https://github.com/triplea-game/triplea/blob/master/config/checkstyle/checkstyle.xml)
```

Comment:
we probably should assume that a dev has cloned the codebase and therefore would not need to download the file. A link to the file is still not bad though as it gives context where to find the checkstyle config file.

Reply:
I thought the link would be handy if someone wanted to set IntelliJ all the way up before cloning/forking the repo.

Reply:
@DanVanAtta Was there anything you wanted from me besides having that knowledge? I think that is what you were after, so I am going to unassign myself now.  Feel free to put me back on it.

Reply:
FWIW, I generally like to see questions about code be answered by updating the code.

IE: 
- Reviewer: "This area of the code is unclear"
- Author: "I have updated the code here, '<commit sha>' to try and clarify, let me know if that covers it?"
- Reviewer: "Yes, the update clears up the question, thank you".

In other words, a reviewer is simulating a future developer that is reading through code. The questions/answers in a PR are of very low value because they are ephemeral. It is a sad day when you have to start looking through PRs to understand the code as written.

With that all aside, let's optimize our setup instructions for someone getting setup. I don't know if there is any value in setting up a project-specific config without having the project checked out. The easiest way to start is to clone, then open IDEA, "open" > select the 'build.gradle' file, then IDEA will automatically add triplea as a gradle roject. From there, the checkstyle config is project specific, specific to TripleA and AFAIK you really need to have the project set up to configure that.

Reply:
First, did i do this new commit correctly, and second, if it worked, is that kind of what you were thinking?

Reply:
Looks good, close enough :+1: